packages/palette/src/elements/BaseTabs/BaseTab.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import styled, { css } from "styled-components"
+import { color } from "../../helpers"
+import { Text, TextProps } from "../Text"
+
+/**
+ * Utilize as="a" or as={Component} to alter functionality
+ */
+export type BaseTabProps = TextProps & {
+  active?: boolean
+} & (// FIXME: Should be able to remove this typing once styled-components is updated
+  // Default:
+  | {}
+    // When displaying as an anchor tag, accept anchor props:
+    | ({ as: "a" } & (React.AnchorHTMLAttributes<HTMLAnchorElement>))
+    // Cop-out: if you pass as with a complex component, loosen up the types:
+    | ({ as: React.ReactElement<unknown> } & Record<string, unknown>))
```

Comment:
`as` seems straight up broken in our version of styled components. Would really like to push forward with the upgrade. Is that a ticket somewhere?